<launch>
	<node pkg="behaviour_planner" type="planner_node.py" name="behaviour_planner_node" output="screen">
	   <param name="behaviourBias" value="2" />
	   <param name="goalBias" value="0.1" />
	</node>
</launch>